medication statements/README.md

```diff
@@ -0,0 +1,363 @@
+| ehrSupplyAuthorise / quantity / translation \[@value]                            | `medicationRequest.dispenseRequest.quantity.value` when present, otherwise a default value of `"1"` is used                                                                                                                                                                                                                                                                           |
+| ehrSupplyAuthorise / quantity / translation / originalText                       | `medicationRequest.dispenseRequest.quantity.unit`, `medicationRequest.dispenseRequest.extension[0].value`<sup>6</sup>, `medicationRequest.dispenseRequest.quantity.extension[0].value`<sup>7</sup><sup>8</sup>                                                                                                                                                                        |
+| ehrSupplyAuthorise / pertinentInformation \[@typeCode]                           | fixed value = `"PERT"`                                                                                                                                                                                                                                                                                                                                                                |
+| ehrSupplyAuthorise / pertinentInformation / pertinentSupplyAnnotation / text     | `medicationRequest.dosageInstruction[0].patientInstruction` prefixed by `"Patient Instruction: "` & `medicationRequest.dispenseRequest.expectedSupplyDuration.value` prefixed by `"Expected Supply Duration"` and suffixed with `medicationRequest.dispenseRequest.expectedSupplyDuration.code` displayed as a unit of time & all `medicationRequest.note.text` prefixed by `"Notes"` |
+
+1. The `codableConcept` from `medicationStatement.extension[0].value` from the `ehrExtract` where `medicationStatement.basedOn` contains a reference to `medicationRequest.id` and `medicationStatement.extension[0].url` = `"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-PrescribingAgency-1"` is used to generate the values.
+2. When code.coding`.system` = `"https://fhir.nhs.uk/STU3/CodeSystem/CareConnect-PrescribingAgency-1"` and `code.coding.code` = `"prescribed-at-gp-practice"` or `"prescribed-by-previous-practice"` then `@code="394823007"` and `@displayName="NHS Prescription"`<br> When `code.coding.code` = `"prescribed-by-another-organisation"` then `@code="394828003"` and `@displayName="Prescription by another organisation"`. In both cases `@codeSystem="2.16.840.1.113883.2.1.3.2.4.15"`
```

Comment:
```suggestion
2. When `code.coding.system` = `"https://fhir.nhs.uk/STU3/CodeSystem/CareConnect-PrescribingAgency-1"` and `code.coding.code` = `"prescribed-at-gp-practice"` or `"prescribed-by-previous-practice"` then `@code="394823007"` and `@displayName="NHS Prescription"`<br> When `code.coding.code` = `"prescribed-by-another-organisation"` then `@code="394828003"` and `@displayName="Prescription by another organisation"`. In both cases `@codeSystem="2.16.840.1.113883.2.1.3.2.4.15"`
```

Reply:
Although, actually I'm a bit confused by this statement in general.

Reply:
Have hopefully cleared this section up and explained it better